core/chains/evm/txmgr/attempts.go

```diff
@@ -75,11 +76,41 @@ func (c *evmTxAttemptBuilder) NewBumpTxAttempt(ctx context.Context, etx Tx, prev
 	if err != nil {
 		return attempt, bumpedFee, bumpedFeeLimit, true, pkgerrors.Wrap(err, "failed to bump fee") // estimator errors are retryable
 	}
-
+	// If transaction's previous attempt is marked for purge, ensure the new bumped attempt also sends empty payload, 0 value, and LimitDefault as fee limit
+	if previousAttempt.IsPurgeAttempt {
+		etx.EncodedPayload = []byte{}
+		etx.Value = *big.NewInt(0)
+		bumpedFeeLimit = c.feeConfig.LimitDefault()
+	}
 	attempt, retryable, err = c.NewCustomTxAttempt(ctx, etx, bumpedFee, bumpedFeeLimit, previousAttempt.TxType, lggr)
+	// If transaction's previous attempt is marked for purge, ensure the new bumped attempt is also marked for purge
+	if previousAttempt.IsPurgeAttempt {
+		attempt.IsPurgeAttempt = true
+	}
 	return attempt, bumpedFee, bumpedFeeLimit, retryable, err
 }
 
+func (c *evmTxAttemptBuilder) NewPurgeTxAttempt(ctx context.Context, etx Tx, lggr logger.Logger) (attempt TxAttempt, err error) {
+	// Use the LimitDefault since this is an empty tx
+	gasLimit := c.feeConfig.LimitDefault()
+	// Transactions being purged will always have a previous attempt since it had to have been broadcasted before at least once
+	previousAttempt := etx.TxAttempts[0]
+	keySpecificMaxGasPriceWei := c.feeConfig.PriceMaxKey(etx.FromAddress)
+	bumpedFee, _, err := c.EvmFeeEstimator.BumpFee(ctx, previousAttempt.TxFee, etx.FeeLimit, keySpecificMaxGasPriceWei, newEvmPriorAttempts(etx.TxAttempts))
```

Comment:
Based on my other comment, do you think it is ok if we just not use bumped gas here, but just get a fresh estimate from chain.
If this will be underpriced, then our Confirmer already has code to keep bumping in a loop until Tx is accepted.